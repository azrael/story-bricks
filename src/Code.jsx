import React from 'react';
import s from './story.css';

const Code = ({ children }) => <code className={s.code}>
    {children}
</code>;

Code.displayName = 'Code';

export default Code;
