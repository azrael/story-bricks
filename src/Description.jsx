import React from 'react';
import s from './story.css';

const Description = ({ children }) => <div className={s.description}>
    {children}
</div>;

Description.displayName = 'Description';

export default Description;
