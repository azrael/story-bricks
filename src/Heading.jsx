import React from 'react';
import s from './story.css';

const Heading = ({ children }) => <div className={s.heading}>
    {children}
</div>;

Heading.displayName = 'Heading';

export default Heading;
