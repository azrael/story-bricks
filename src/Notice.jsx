import React from 'react';
import s from './story.css';

const Notice = ({ children }) => <div className={s.notice}>
    {children}
</div>;

Notice.displayName = 'Notice';

export default Notice;
