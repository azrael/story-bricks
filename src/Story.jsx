import React from 'react';
import s from './story.css';

const Story = ({ children }) => <div className={s.container}>
    {children}
</div>;

Story.displayName = 'Story';

export default Story;
