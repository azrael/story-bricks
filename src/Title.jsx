import React from 'react';
import s from './story.css';

const Title = ({ children }) => <div className={s.title}>
    {children}
</div>;

Title.displayName = 'Title';

export default Title;
