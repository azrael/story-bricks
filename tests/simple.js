import React from 'react';
import renderer from 'react-test-renderer';
import Code from '../src/Code.jsx';
import Description from '../src/Description.jsx';
import Heading from '../src/Heading.jsx';
import Notice from '../src/Notice.jsx';
import Story from '../src/Story.jsx';
import Title from '../src/Title.jsx';

it('Code renders correctly', () => {
    const tree = renderer
        .create(<Code>test</Code>)
        .toJSON();

    expect(tree).toMatchSnapshot();
});

it('Description renders correctly', () => {
    const tree = renderer
        .create(<Description>test</Description>)
        .toJSON();

    expect(tree).toMatchSnapshot();
});

it('Heading renders correctly', () => {
    const tree = renderer
        .create(<Heading>test</Heading>)
        .toJSON();

    expect(tree).toMatchSnapshot();
});

it('Notice renders correctly', () => {
    const tree = renderer
        .create(<Notice>test</Notice>)
        .toJSON();

    expect(tree).toMatchSnapshot();
});

it('Story renders correctly', () => {
    const tree = renderer
        .create(<Story>test</Story>)
        .toJSON();

    expect(tree).toMatchSnapshot();
});

it('Title renders correctly', () => {
    const tree = renderer
        .create(<Title>test</Title>)
        .toJSON();

    expect(tree).toMatchSnapshot();
});
