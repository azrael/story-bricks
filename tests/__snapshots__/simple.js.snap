// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Code renders correctly 1`] = `
<code
  className="code"
>
  test
</code>
`;

exports[`Description renders correctly 1`] = `
<div
  className="description"
>
  test
</div>
`;

exports[`Heading renders correctly 1`] = `
<div
  className="heading"
>
  test
</div>
`;

exports[`Notice renders correctly 1`] = `
<div
  className="notice"
>
  test
</div>
`;

exports[`Story renders correctly 1`] = `
<div
  className="container"
>
  test
</div>
`;

exports[`Title renders correctly 1`] = `
<div
  className="title"
>
  test
</div>
`;
